pages/glossary/browsing-context-related-documents.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Set of browsing context related documents
+key: browsing-context-related-documents
+---
+
+Two [documents](https://dom.spec.whatwg.org/#concept-document) are *browsing context related* if they are [fully active](https://html.spec.whatwg.org/#fully-active); and their respective [browsing context](https://html.spec.whatwg.org/#browsing-context) have a common [ancestor browsing context](https://html.spec.whatwg.org/#ancestor-browsing-context).

```

Comment:
Please use W3C documentation. The WHATWG spec is a living standard. These links, and the contents of them can change.

Reply:
@WilcoFiers The HTML and DOM specifications are no longer published by the W3C, please see https://www.w3.org/2019/04/WHATWG-W3C-MOU.html.

Reply:
@WilcoFiers @kasperisager here is what I'm understanding of the situation (and your respective positions):
* W3C stopped *writing* HTML/DOM specs, WHATWG starts writing them as Living Standard.
* W3C will nonetheless *publish* these as REC sometimes.
* HTML 5.2 is the latest *stable* standard (W3C REC) and does have a stable URL at https://www.w3.org/TR/html52/ while the editor draft of the "next" version at https://www.w3.org/TR/html/ redirects to WHATWG Living Standard.
* It is likely that at some point in the future, the WHATWG LS gets frozen into HTML 5.3 and that would get its own (non-redirect) URL at https://www.w3.org/TR/html53/ (that's how I understand the fact that W3C will *publish* as REC the result of WHATWG work). WHATWG (and https://www.w3.org/TR/html/) will still hold the newer Living Standard.

---
When it comes to us, the question is what we should link. @WilcoFiers says that we should always link a frozen version (*i.e.* 5.2) because the LS can change without notice and breaks our rules. @kasperisager says that we should link the latest version (*i.e.* WHATWG) because (I guess) if browsers follow it and it differs from 5.2, then our rules would be completely outdated.

I do not have a strong opinion on that choice. 

Recently (since I joined), we've been moving links from https://www.w3.org/TR/html/ to WHATWG. It is worth noting that links to https://www.w3.org/TR/html/ were anyway "living" since that was not a frozen version number.

Whatever the choice we make (5.2 or WHATWG), the important thing (for me) is that we are coherent among all our rules. Currently the choice is more toward WHATWG (20 occurrences vs 4 for html52), but this is largely due to the move that was recently made (notably by @annethyme , @jkodu and me). So it is in no way a clear decision.

---
I think we (likely meaning @WilcoFiers and @kasperisager since you seem to have a strong opinion about it) agree on which document we should cite and stick to it (and update the other refs). This ripples through all rules and several current PR (eg, #853) and so should be settled soonish. Probably a good agenda item for ACT-R CG.

As I said, I don't care which way we go for citing (5.2 or WHATWG), but I want to know which way we go so I can do it correctly…

Reply:
Similarly to #853, I will keep the current citations until #879 is resolved and we have a global PR fixing everything.